listenbrainz/spark/spark_reader.py

```diff
@@ -87,7 +87,7 @@ def process_response(self, response):
         try:
             response_handler(response)
         except Exception as e:
-            current_app.logger.error('Error in the response handler: %s, data: %s %'
+            current_app.logger.error('Error in the response handler: %s, data: %s' %
```

Comment:
remember that logger methods will do string interpolation automatically anyway, so you should be able to do `logger.error('message %s', var, exc_info=True)`